hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd

```diff
@@ -1017,6 +1017,18 @@
             </xs:appinfo>
         </xs:annotation>
     </xs:element>
+    <xs:element name="replicatedMap" type="hazelcast-type">
+        <xs:annotation>
+            <xs:documentation>
+                Retrive an hazelcast ReplicatedMap instance
```

Comment:
http://editingandwritingservices.com/a-or-an-before-words-beginning-with-h/

I think @Serdaro can help us on that. :)